scylla/README.md

```diff
@@ -20,6 +20,35 @@ The Scylla check is included in the [Datadog Agent][2] package. No additional in
 
 2. [Restart the Agent][4].
 
+##### Log collection
+
+Scylla has different modes of outputting logs depending on the environment its running in.  See the [Scylla documentation][8] for more specifics on how the application generates logs.
```

Comment:
When I was working on this section for flink, I noticed that most other integrations have a warning such as `Available for Agent >6.0` below the log header. I'm not sure if it's necessary to include but it could be helpful.

Reply:
I don't think we need that anymore as we've deprecated Agent 5

Reply:
Ok sounds good, I can delete the line from mine.